auth/services/selfsigned/validator.go

```diff
@@ -135,6 +139,22 @@ func (v validator) verifyVP(vp vc.VerifiablePresentation, validAt *time.Time) (c
 		return
 	}
 
+	// #2428: NutsEmployeeCredential trust is derived from the fact that the issuer has a trusted NutsOrganizationCredential
+	searchTerms := []vcr.SearchTerm{
+		{IRIPath: jsonld.CredentialSubjectPath, Value: vc.Issuer.String()},
```

Comment:
Better to search for the subject here, since the subject is the `Organisation`. Yes, the credential is self-issued, but this is more a business rule.

Reply:
I think that property is not indexed, so search might be impacted. Actually, you'd want to assert both, but since the presentation and credential specify they must be the same, it doesnt really matter?

Reply:
What I mean is, the value of the jsonld.CredentialSubjectPath should be the credentialSubject ID and not the Issuer. Since the subject of the EmployeeID credential is the same organisation as the nutsOrgCredential subject ID.